### PR TITLE
[about] Fix photo caption as rendered in the generated HTML

### PR DESCRIPTION
**Pull request by romen**
Someone pointed out to me the recent updates to the _About Us_ page, and I noticed that the rendered HTML offers a poor caption for the photograph:

image

This fixes it reusing the same text that ends up in the _alt_ description as the proper caption.

### CODE REVIEW

File: about.md [hunk 1]
```diff
@@ -10,7 +10,7 @@ security and privacy tools. Our dedicated team is at the helm of the OpenSSL
 project, ensuring secure digital interactions. Here, you'll meet the individuals
 who are collectively shaping a safer, privacy-respecting digital future.
 
-![team](/img/faces/openssl2023.png "Some of the OpenSSL team in 2023")
+![Some of the OpenSSL team in 2023](/img/faces/openssl2023.png "Some of the OpenSSL team in 2023")
```
Comment (paulidale) [inline about.md]: Maybe include a "listed below in alphabetic order".
I.e. to indicate that there is no relationship to the left to right in the photograph.

**Review by mattcaswell — APPROVED** [review]
I'm ok with this as-is. If we want to make the update suggested by @paulidale, that would also be ok.

File: about.md (hunk 1 above)
Comment (romen) [reply about.md]: Good point, done!

**Review by mattcaswell — APPROVED** [review]

**Comment by romen** [comment]
I am not familiar with the process followed to merge PRs in this repo.
I suppose I don't have access rights here, so could whoever merges this take care of squashing the commits once everything is approved, please?

**Comment by arapov** [comment]
See #413, I think it is a better way to fix it. Also, I like mentioning "in alphabetic order" at all.

NAK for this PR from me.

**Comment by romen** [comment]
I like Anton's solution more! Closing!

**Pull request closed without merging**